Makefile

```diff
@@ -1,8 +1,32 @@
 SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
 ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+DB_DEV = core
+RAFT_DEV = $(HOME)/.chaincore/raft
```

Comment:
@tessr Nice catch, I was a little caught up on that myself. I think I can do one better and read CHAIN_CORE_HOME if available:

```
RAFT_DEV = $(or $(CHAIN_CORE_HOME), $(HOME)/.chaincore)/raft
```